account_statement_import_paypal/models/account_statement_import_paypal_parser.py

```diff
@@ -207,9 +206,9 @@ def _convert_line_to_transactions(self, line):
             note += " (%s)" % payer_email
 
         unique_import_id = f"{transaction_id}-{int(timestamp.timestamp())}"
-        name = (invoice or details or description or "",)
+        name = f"invoice or details or description or ''-{transaction_id}"
```

Comment:
What is this? 

I think the correct code is:

name = invoice or details or description or "